source/evp_override.c

```diff
@@ -706,6 +707,35 @@ const EVP_MD *EVP_sha512() {
     return &md;
 }
 
+EVP_MD *EVP_MD_fetch(OSSL_LIB_CTX *ctx, const char *algorithm, const char *properties) {
+    // Assume this method is always called without a library context
```

Comment:
Should this be:
```suggestion
    // Assume this method is never called without a library context
```
?

Reply:
... as written, yes. But actually I meant to assume the ctx is always NULL. s2n-tls always calls it with NULL. Let me investigate why this is working. s2n-tls's proofs wouldn't have cared about my other dumb mistakes, but this one they really should have cared about.